src/libvol2bird/libvol2bird.c

```diff
@@ -4867,6 +5051,25 @@ int vol2birdLoadConfig(vol2bird_t* alldata, const char* optionsFile) {
 
 #endif
 
+char* get_radar_name(char* source) {
```

Comment:
change to `int get_radar_name(char* source, char* radarName)`, copying radar name from source into misc.radarName field. Output int can indicate of copy was succesful